apps/web/src/contexts/globalLayout/GlobalNavbar/HomeNavbar/HomeNavbar.tsx

```diff
@@ -94,7 +98,14 @@ export function HomeNavbar({ queryRef }: Props) {
       </NavbarCenterContent>
 
       {/* Strictly here to keep spacing consistent */}
-      <NavbarRightContent>{isLoggedIn ? null : <SignInButton />}</NavbarRightContent>
+      <NavbarRightContent>
+        {isLoggedIn ? null : (
+          <HStack gap={8} align="center">
+            <SignInButton />
+            {!isMobile && <SignUpButton />}
+          </HStack>
+        )}
+      </NavbarRightContent>
```

Comment:
nit: could we leave a comment here explaining why we're disabling signup on mobile

Reply:
added! good call